### ISSUE TITLE
Perform audit of bb (prod) ARKs on staging

### ISSUE THREAD

**Issue by gamontoya** [ucsdlib/damspas#347]
### Descriptive summary

An audit needs for assigned ARKs on staging; specifically looking for any bb-ARKs used on staging. 

### Rationale

Ryan is working on a collection on staging that has a **bb-ARK** (https://librarytest.ucsd.edu/dc/collection/bb8158593z). It should be using a **bd-ARK** and the question seems to be, _"How did this happen?"_

- We need to ensure that DAMS Manager did not mint a production ARK from the staging environment.
- Was this record a result of a sync at some point to get prod data into staging?

**Comment by VivianChu** [comment]
@gamontoya - I confirmed with Ron that the properties file for damsManager in staging is configured to use bd ark minter. I also created a test collection record in staging and it used a bd ark.
https://librarytest.ucsd.edu/dc/collection/bd8720339f
The collection record above (https://librarytest.ucsd.edu/dc/collection/bb8158593z) was also in production  https://library.ucsd.edu/dc/collection/bb8158593z
I think this record was a result of a sync to get prod data into staging.
The last time we did the data sync was on July 2017
https://lib-jira.ucsd.edu:8443/browse/IO-47
In staging , 128 collection records have bb ark and 54 collection records have bd ark.
110927 objects have bb ark and 4066 have bd ark.
